web/themes/custom/cloudy/scss/components/_card.scss

```diff
@@ -1,19 +1,18 @@
 .node--type-city-service {
-  .card:not(:first-child) {
-    max-width: calc(50% - 30px);
-  }
   .card {
-    color: red;
-    flex-basis: calc(50% - 30px);
-      @include media-breakpoint-up(sm) {
-        margin-bottom: $card-deck-margin;
-      }
-      @include media-breakpoint-only(md) {
-        flex-basis: 100%;
-        flex-direction: row;
-      }
-    @include media-breakpoint-up(lg) {
-      flex-wrap: wrap;
+    &:first-child {
+      max-width: 100%;
+    }
+
+    @include media-breakpoint-up(sm) {
+      flex-basis: calc(50% - 30px);
+      max-width: calc(50% - 30px);
+      margin-bottom: $card-deck-margin;
```

Comment:
Consider multiplying this by two. The margin between cards is asynchronous horizontally and vertically.  Alternatively, I like trimming the margins down.  I played with making the card margin .5rem and liked the results.  You'd need to override the variable carefully, after looking at bootstrap's definition for it, as it is related to another card variable.